### DISCUSSION TITLE
Some screens I have acquired

### DISCUSSION THREAD

**Discussion by CaptainChristms** [ginbot86/ColorLCDVape-RE#6] [Q&A]
These are the screens I have acquired from 2 different types of vapes. One being one of the Raz line, and the other being from the Rama tl16000 (with Bluetooth capability). Oh, and one screen from an old broken camera. I can’t stand having them sit around any longer. I have to find a use for them, but I need help, which is why I’ve come to you! I have a tiny gaming console called the “FunKey S” that I ended up breaking the screen on, and the vape screens are the perfect size fit for replacement, but unfortunately, the connectors don’t fit and I have no idea how to make it fit/hardwire it in. I’d also love to change it from micro usb to type-c.
![IMG_8540](https://github.com/user-attachments/assets/710ee990-c6a6-492b-acf3-9aaccce102be)
![IMG_8541](https://github.com/user-attachments/assets/666819c7-6f56-4863-a03f-2cb2e11c80a8)
![IMG_8488](https://github.com/user-attachments/assets/88aa23b3-2b36-41f3-b074-17861494939d)
![IMG_8487](https://github.com/user-attachments/assets/f76b78ca-5cf9-47c5-9158-00c7485c2e92)
![image](https://github.com/user-attachments/assets/187a4384-6d85-406c-9dc3-18437f2c475c)

**Comment by ginbot86** [comment]
Neat! Unfortunately the vape screens' resolutions won't match with the FunKey S, as per [their documentation](https://doc.funkey-project.com/developer_guide/hardware_reference/screen/). That screen is based on an [ST7789V](https://newhavendisplay.com/content/datasheets/ST7789V.pdf) over SPI, thankfully.

The camera screen will likely be harder to repurpose. It probably uses an RGB dot-clock/raw digital interface,

USB-C conversions might work but space is limited on that board to do so. That reminds me, I should write up the blog post where I converted the micro-USB host port on an Acer Aspire Switch 10 tablet into a USB-C one... 🤔

**Reply by CaptainChristms** [reply]
I actually just found the pcb design for the funkey S! Do you think if I took that design and redesigned it and made it a bit bigger with connections that could take those screens and replaced the the micro usb with a type c, it would work?

**Reply by CaptainChristms** [reply]
I need to find someone who will design it, because I have no idea how to do it myself 🤣

**Reply by ginbot86** [reply]
Theoretically yes, and that's one great thing about open source hardware! The other issue is changing the software to handle the reduced display resolution...
